modules/material/material.resolver.js

```diff
@@ -16,6 +16,16 @@ const materialQuery = {
       message: MATERIAL_NOT_FOUND,
     };
   },
+  getMaterialColorByCode: async (parent, args) => {
+    const material = await materialService.getMaterialColorByCode(args.code);
+    if (material[0].colors[0]) {
+      return material[0].colors[0];
+    }
+    return {
+      statusCode: 404,
+      message: COLOR_NOT_FOUND,
```

Comment:
Try to use the same approach to the implementation of the method, move error handling to services as you do in other methods

Reply:
Fixed